src/components/confirmation.js

```diff
@@ -0,0 +1,12 @@
+// import React from 'react';
```

Comment:
Aren't you using this component anymore? Why did you comment it? Did everyone create their component? If you are not using it, can you delete it?